### PR TITLE
feat: Add function hasPolicies

### PR DESCRIPTION
**Pull request by Edmond-J-A**
Signed-off-by: Edmond <edomondja@gmail.com>

### CODE REVIEW

File: src/model/Policy.lua [hunk 1]
```diff
@@ -172,6 +172,24 @@ function Policy:hasPolicy(sec, ptype, rule)
...
+     * hasPolicies determines whether a model has any of the specified policies.
+     *
+     * @param sec the section, "p" or "g".
+     * @param ptype the policy type, "p", "p2", .. or "g", "g2", ..
+     * @param rules the policies rules.
+     * @return whether one is found we return true.
+]]
+function Policy:hasPolicies(sec, ptype, rules)
+    for i = 1, #rules do
+        if self.model.hasPolicy(sec, ptype, rules[i]) then
```
Comment (rushitote) [inline src/model/Policy.lua:185]: It should be `self:hasPolicy(sec, ptype, rules[i])`.

**Review by rushitote — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@rushitote

**Review by rushitote — APPROVED** [review]
lgtm

File: src/model/Policy.lua [hunk 2]
```diff
@@ -172,6 +172,23 @@ function Policy:hasPolicy(sec, ptype, rule)
...
+]]
+function Policy:hasPolicies(sec, ptype, rules)
+    for i = 1, #rules do
+        if self:hasPolicy(sec, ptype, rules[i]) then
+            return true
+		end
+    end
+    return false
+end
+
```
Comment (techoner) [inline src/model/Policy.lua:191]: @Edmond-J-A Format your code and don’t forget to add unit tests .

**Review by techoner — CHANGES_REQUESTED** [review]

File: src/model/Policy.lua (hunk 2 above)
Comment (Edmond-J-A) [reply src/model/Policy.lua:191]: Got it，unit tests has been added.

**Pull request merged**